lib/dynamo/dynamoDbClient.js

```diff
@@ -0,0 +1,69 @@
+/*
+ *  == BSD2 LICENSE ==
+ */
+
+'use strict';
+
+var _ = require('lodash');
+var log = require('../log.js')('dynamoDbClient.js');
+var AWS = require('aws-sdk');
+
+module.exports = function(awsconfig, tablename) {
+  AWS.config.update(awsconfig);
+  var db = new AWS.DynamoDB();
+
+  return {
+    submit: function(user, eventName, event) {
+      // fill in the required parameters
+      var parms = {
+        _p: user,
+        _n: eventName
+      };
+      // do the timestamp if it's specified
+      var args = _.clone(event);
+      log.info('metric[%s], user[%s]: %j', parms._n, parms._p, event);
+
+      if (args.timestamp) {
+        parms._t = args.timestamp;
+      } else {
+        parms._t = Date.now();
```

Comment:
No, actually -- date needs to be an integer value in this case, for compatibility with the KISSMetrics logs.